app/domain/operations/transmittable/generate_request_objects.rb

```diff
@@ -0,0 +1,138 @@
+# frozen_string_literal: true
+
+module Operations
```

Comment:
@jacobkagon I recommend you to nest this under the SugarCRM namespace as this is not a generic operation.